.docs/user-guide/storage-providers.md

```diff
@@ -45,8 +45,8 @@ The following items are configurable specific to this driver.
 
  * `volumePath` represents the location under `/ifs/volumes` to allow volumes to
    be created and removed.
- * `nfsHost` is the configurable host used when mounting exports
- * `dataSubnet` is the subnet the REX-Ray driver is running on
+ * `nfsHost` is the configurable NFS server hostname or IP (often a SmartConnect name) used when mounting exports
+ * `dataSubnet` is the subnet the REX-Ray driver is running on.  This is used for the NFS export host ACLs.
```

Comment:
Hi @spiegela,

I also learned in school to use two spaces after a period, but that rule has since been changed thanks to the interwebs. The [convention](https://en.wikipedia.org/wiki/Sentence_spacing#Typography) is now a single space after a period. Please update. Thank you. Also, please limit the column width to 80 characters. Thanks again.